contrib/terraform/gcp/README.md

```diff
@@ -0,0 +1,165 @@
+## Kubernetes on GCP with Terraform
+
+
+Provision a Kubernetes cluster on GCP using [Terraform](https://www.terraform.io) and [Kubespray](https://github.com/kubernetes-sigs/kubespray.git)
+
+![Workflow](Workflow.svg)
+
+**Overview:**
+
+This project will create:
+
+* Creation of **Kubernetes master** Instance template and Instance Group (1 & 2).The Compute instances are created in different zones.
+* Creation of **Kubernetes etcd** Instance template and Instance Group (3 & 4).The Compute instances are created in different zones.
+* Creation of **Kubernetes worker nodes/minions** Instance template and Instance Group (5 & 6).The Compute instances are created in different zones.
```

Comment:
i believe we don't call nodes anymore as minions, this should be removed everywhere to not confuse people